projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.service.spec.ts

```diff
@@ -102,25 +123,43 @@ describe('MerchandisingCarouselComponentService', () => {
     );
     expectedMerchandisingCarouselModelMetadata.set(
       'slots',
-      mockMerchandisingProducts.products.length.toString()
+      mockStrategyProducts.products.length.toString()
     );
     expectedMerchandisingCarouselModelMetadata.set('id', mockComponentData.uid);
 
+    const expectedMerchandisingCarouselModelProducts: MerchandisingProduct[] = mockStrategyProducts.products.map(
+      (strategyProduct, index) => {
+        const merchandisingProductMetadata = new Map<string, string>(
+          Object.entries(strategyProduct.metadata)
+        );
+        merchandisingProductMetadata.set('id', strategyProduct.id);
+        merchandisingProductMetadata.set('slot', (index + 1).toString());
```

Comment:
just trying to say consistent with this one but the data that gets scraped has a key called slots or slot?